spec/commands/process_raw_event_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "ProcessRawEventCommand" do
+  let(:application) { create :application }
+  let(:raw_event)   { create(:raw_event) }
+
+  it "Process Raw Evnt Command work and store final data" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.